packages/components/src/VirtualizedList/ListGrid/ListGrid.component.js

```diff
@@ -69,11 +73,13 @@ ListGrid.propTypes = {
 	rowHeight: PropTypes.number,
 	rowRenderer: PropTypes.func,
 	width: PropTypes.number,
+	t: PropTypes.func,
 };
 
 
 ListGrid.defaultProps = {
 	rowHeight: 135,
+	t: getDefaultTranslate,
```

Comment:
This component is not supposed to be used outside of VirtualizedList. So the default `t()` is not necessary. We can consider that it is required, and it is passed by `virtualizedList` component. 

In practice this is already the ccase, your default is always overriden by the one from `VirtualizedList`